3rdparty/openssl/CMakeLists.txt

```diff
@@ -354,16 +359,51 @@ macro (append_generate_command LIST FILE_NAME SOURCE_DIR DEST_DIR)
     ${DEST_DIR}/${FILE_NAME}.s)
 endmacro ()
 
+macro (append_generate_3_command LIST FILE_NAME SOURCE_DIR DEST_DIR)
```

Comment:
Maybe we rename this and the above macro, as this is no longer really generating a file. This could be called "append_copy_assembly_command" and the above macro could be "append_generate_assembly_command"